src/Dumplie/Domain/Customer/CartItem.php

```diff
@@ -36,6 +36,14 @@ public function __construct(Product $product, int $quantity)
     }
 
     /**
+     * @return Product
+     */
+    public function product() : Product
```

Comment:
I think I already wrote that, but there is no need to store `Product` inside `CartItem`.
Storing products `SKU` is enough.